GripOpGras2.Client/Features/CreateRation/IRationAlgorithm.cs

```diff
@@ -1,10 +1,15 @@
 ﻿using GripOpGras2.Domain;
+using GripOpGras2.Domain.FeedProducts;
 
 namespace GripOpGras2.Client.Features.CreateRation
 {
 	public interface IRationAlgorithm
 	{
-		public Task<FeedRation> CreateRationAsync(IReadOnlyList<Roughage> roughages, Herd herd, float totalGrassIntake,
-			GrazingActivity grazingActivity, MilkProductionAnalysis milkProductionAnalysis);
+		/// <summary>
+		/// Calculates the ration for the herd.
+		/// </summary>
+		/// <param name="totalGrassIntake">The total grass intake of the herd in kg dm.</param>
```

Comment:
Ik heb het volgnde, moet dit er niet staan?: 

		/// <summary>
		/// Calculates the ration for the herd.
		/// </summary>
		/// <param name="feedProducts"></param>
		/// <param name="herd"></param>
		/// <param name="totalGrassIntake">The total grass intake of the herd in kg dm.</param>
		/// <param name="milkProductionAnalysis"></param>
		/// <param name="grazingActivity"></param>

Reply:
Nee dat hoeft niet

Reply:
Wanneer je gras als "FeedProduct" neerzet, dan maak je het mogelijk dat een rantsoen meerdere gras producten kan bevatten. Tot nu toe hebben wij het Marcia telkens gehad over dat je gras hebt, en je dit bijvult met producten. Lijkt me niet handig om deze aanpak nu nog te veranderen.

Reply:
kaykaykay, dan snap ik miss nog nit hlmaal wat ht stukje 'param name' doet